components/BluePrint/environment/Visibility.tsx

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { styleObj } from '../../../styles/styles'
+import { useAppContext } from '../../../context/AppProvider'
+import * as uiTypes from '../../../types/blue-print'
+import * as utils from '../../../components/DataTable/TableBody/utils'
+
+function Visibility() {
+  return <div />
+}
```

Comment:
This component is just returning an empty div. Let's remove this component (file) as it doesn't seem to be doing much.